services/visual-studio-marketplace/visual-studio-marketplace-base.js

```diff
@@ -27,6 +27,8 @@ const extensionQuerySchema = Joi.object({
                 )
                 .min(1)
                 .required(),
+              releaseDate: Joi.string().required(),
+              lastUpdated: Joi.string().required(),
```

Comment:
Are both of these field guaranteed to always be there for all packages, and never be null or undefined? `releaseDate` feels like a safe bet, but want to make sure that `lastUpdated` is there immediately after the initial publish and not just null until the first _update_

Reply:
Yes, `lastUpdated` is there right after the initial publish. You can try it out using this extension I found with only 1 release: `tscpp.node-dependencies`.